utils/gql.py

```diff
@@ -13,23 +13,58 @@ class GqlApiError(Exception):
     pass
 
 
+class GqlApiIntegrationNotFound(Exception):
+    def __init__(self, integration):
+        super().__init__(f"integration not found: {integration}")
```

Comment:
can we add a hint here about what can be done? where is the integration missing from and what should a user do?